Brain/Brain.ino

```diff
@@ -15,40 +15,37 @@
 #define RFSerial Serial2
 #define GPSSerial Serial3
 
-
-// within loop state variables
 int board_address = 0;
-byte sensor_id = 0;
+int sensor_id = 0;
 uint8_t val_index = 0;
 String command = "";
 
 /*
  * Array of all sensors we would like to get data from.
  */
-sensorInfo all_ids[8] = {
-  {"Low Pressure",8,0,1,1,NULL} //example
+sensorInfo all_ids[11] = {
+  // local sensors
+   {"LOX Injector Low Pressure",  -1, -1, 1, 1, &(Ducers::readLOXInjectorPressure)} //example
 };
 
-sensorInfo sensor = {"",0,0,0,0, NULL};
-
-int numSensors = 1;
+sensorInfo sensor = {"", 0, 0, 0, 0, NULL};
```

Comment:
keep this variable in. We'll need it.

Reply:
For what? I didn't see it in use so I didn't leave it in

Reply:
hmm... I guess you're right. I thought I was using it somewhere